mypy/newsemanal/semanal.py

```diff
@@ -2396,16 +2410,22 @@ def check_and_set_up_type_alias(self, s: AssignmentStmt) -> bool:
 
     def analyze_lvalue(self, lval: Lvalue, nested: bool = False,
                        explicit_type: bool = False,
-                       is_final: bool = False) -> None:
+                       is_final: bool = False,
+                       escape_comprehensions: bool = False) -> None:
         """Analyze an lvalue or assignment target.
 
         Args:
             lval: The target lvalue
             nested: If true, the lvalue is within a tuple or list lvalue expression
             explicit_type: Assignment has type annotation
+            escape_comprehensions: If we are inside a comprehension, set the variable
+            in the enclosing scope instead. This implements
```

Comment:
Style nit: indent the second and following lines in an argument specification like this:

```
    """
    ...

    Args:
        argument: Something something ...
            explanation continues here ...
```